spec/requests/trucks_request_spec.rb

```diff
@@ -1,10 +1,92 @@
 require 'rails_helper'
 
 describe "Trucks" do
+  # create truck and attributes for create and update
+  let(:truck) { create(:truck) }
+  let(:valid_attributes) do
+    { license_plate: "license_plate", capacity: 1, model_type: truck.model_type, color: "color", service_date: truck.service_date, status: truck.status }
```

Comment:
too long, can we split this across multiple lines please
```suggestion
    { 
      license_plate: "license_plate",
      capacity: 1,
      model_type: truck.model_type,
      color: "color",
      service_date: truck.service_date,
      status: truck.status
    }
```